ConsentViewController/Classes/SourcePointClient.swift

```diff
@@ -130,8 +127,6 @@ class SourcePointClient {
     private let campaignEnv: GDPRCampaignEnv
     private let targetingParams: TargetingParams?
 
-    public var onError: OnError? { didSet { client.defaultOnError = onError } }
```

Comment:
Have you tried using an `unowned` reference instead of removing it?

Reply:
Yes @andresilveirah I have tried with both  **weak and unowned**  but both gives error.